src/main/resources/en/computer.yml

```diff
@@ -58,6 +61,7 @@ en:
         - Packard Bell
         - Razer
         - Samsung
+        - Sony
```

Comment:
Ummm "what-the-diff" has lost it's mind, this is definitely an addition.

Though arguably maybe it should be VAIO?
- https://www.sony.ca/en/electronics/computers/vaio-laptops
- https://en.wikipedia.org/wiki/Vaio
- https://us.vaio.com/

Reply:
What the diff isn't perfect, I'm just trialing it. It's 80% or so correct, and it's more aan FYI than anything.

Having said that: I was surprised too, but then again, I don't think they still sell computers: https://www.sony.com.au/electronics/computers/vaio-laptops

Reply:
Okay. It looks like Sony spun Vaio off and it’s now an independent company.

I’ll just skip it for now. If someone feels stronger about it in the future hopefully they’ve dig into it more 😉

Reply:
So, you're saying the AI was right all along! Scary! :-)